src/MongoDB.Driver/ClusterRegistry.cs

```diff
@@ -88,7 +88,6 @@ private ClusterSettings ConfigureCluster(ClusterSettings settings, ClusterKey cl
                 replicaSetName: clusterKey.ReplicaSetName,
                 maxServerSelectionWaitQueueSize: clusterKey.WaitQueueSize,
                 serverSelectionTimeout: clusterKey.ServerSelectionTimeout,
-                postServerSelector: new LatencyLimitingServerSelector(clusterKey.LocalThreshold),
```

Comment:
I think with the removal of this line we also need the addition of a new line:

    localThreshold: clusterKey.LocalThreshold,

Reply:
Done.